reference/enterprise-api-ref/export-import-api.markdown

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: Import & Export API
+published: true
+tags: [reference, enterprise, API, import, export]
+---
+
+Import & Export API allows to transfer Mission portal data between hubs.
+
+## Get available items to export
+
+This API call provides a list of items available for export. Please note, roles affect access, some items like settings require the admin role for export access.
+
+**URI:** https://hub.example/data_transfer/api/exportItems
+
+**Method:** GET
+
+**Example request (curl):**
+
+```
+curl -k --user <username>:<password> \
+  -X GET \
+  https://hub.example/data_transfer/api/exportItems
+```
+
+**Example response:**
+
+```
+HTTP 200 Ok
+[
+    {
+        "id": "categorizations",
+        "name": "Host categorizations"
+    },
+    {
+        "id": "dashboards",
+        "name": "Dashboards"
+    },
+    {
+        "id": "reports",
+        "name": "Reports"
+    },
+    {
+        "id": "settingsScripts",
+        "name": "Custom notification scripts"
+    },
+    {
+        "id": "users",
+        "name": "Users"
+    },
+    {
+        "id": "settingsRoles",
+        "name": "Roles"
+    },
+    {
+        "id": "settingsPreferences",
+        "name": "Preferences"
+    },
+    {
+        "id": "settingsAuthentication",
+        "name": "LDAP authentication settings"
+    },
+    {
+        "id": "settingsMail",
+        "name": "Mail settings"
+    },
+    {
+        "id": "settingsVCS",
+        "name": "Version control repository"
+    }
+]
+```
+
+**Output:**
+
+* **id**
+    Item id. Use this id in export API call.
+* **name**
+    Name of export item.
+    
+    
+## Export
+
+**URI:** https://hub.example/data_transfer/api/export
+
+**Method:** GET
+
+**Parameters:**
+
+* **item_id** *(array)*
+    Item id to be exported.
+    List of item ids you can obtain through [List of items to export][Import & Export API#Get available items to export]
+        call described below. 
+        
+* **encryptionKey** *(string)*
+    Encryption key to encrypt sensitive data. Please save this key to be able to import the data.
+* **exportOnlyUserItems** *(string)*
+    Export only users item - `true`. Export whole system data - `false`
+
+**Example request (curl):**
+
+```
+curl -k -g --user <username>:<password> \
+  -X GET \
+  'https://hub.example/index.php/data_transfer/api/export?encryptionKey=key&exportOnlyUserItems=true&items[]=categorizations&items[]=dashboards&items[]=settingsAuthentication&items[]=settingsMail'
+
+```
+
+**Example response:**
+
+```
+HTTP 200 Ok
+{
+    "name": "export_12-14-2018_15:19:40.381400.phar",
+}
+```
+
+**Output:**
+
+* **name**
+    Name of export file.
+* **url**
+    Url of export file.
+    
+## Download export file
+
+**URI:** https://hub.example/data_transfer/api/export/:file_name:
+
```

Comment:
this URL should be https://hub.example/data_transfer/api/download/:file_name: 

right?

Reply:
done